run-dljc.sh

```diff
@@ -23,40 +25,32 @@
 #
 # -i infile : read the list of repositories to use from the file $infile. Each
 #             line should contain the (https) url of the git repository on
-#             GitHub and the commit hash to use, separated by whitespace. If the
-#             repository's owner is you (see -u flag), then each line owned by
-#             you must be followed by the original github repository.
-#
-# -c checkers : a comma-separated list of typecheckers to run
+#             GitHub and the commit hash to use, separated by whitespace.
+#             Repositories must be of the form:
+#             https://github.com/username/repository - the script is reliant
+#             on the number of slashes, so excluding https:// is an error.
+#             
+#             If the repository's owner is the user specified by the -u flag,
```

Comment:
Please expand on how and why this could occur.  That is, please expand the experimental methodology in the README file.  I think this is for either fixing bugs or adding annotations that WPI was unable to infer.

Reply:
That is correct. I added an optional fourth step to the instructions that describes that option.